src/types/DTOs.ts

```diff
@@ -1,6 +1,6 @@
 interface CardDTO {
```

Comment:
please change the filename to kebab-case. I refer to this article https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git